perllib/FixMyStreet/App/Controller/Admin/AreaStats.pm

```diff
@@ -0,0 +1,217 @@
+    my $time = $dtf->format_datetime($date);
+
+    my $params = {
+        'me.confirmed' => { '>=', $time },
+    };
+
+    my %area_param = ();
+    if ($c->stash->{area}) {
+        my $area_id = $c->stash->{area_id};
+        $c->stash->{area_name} = $c->stash->{area}->{name};
+        $params->{'problem.areas'} = { like => "%,$area_id,%" };
+        %area_param = (
+            areas => { like => "%,$area_id,%" },
+        );
+    } else {
+        $c->stash->{area_name} = $c->stash->{body}->name;
+    }
+
+    my $comments = $c->model('DB::Comment')->to_body(
+        $c->stash->{body}
+    )->search(
+        $params,
+        {
+            group_by => [ 'problem_state' ],
+            select   => [ 'problem_state', { count => 'me.id' } ],
+            as       => [ qw/state state_count/ ],
+            join     => 'problem'
+        }
+    );
+
+    my $problems = $c->model('DB::Problem')->to_body(
+        $c->stash->{body}
+    )->search(
+        \%area_param,
+        {
+            group_by => [ 'category', 'state' ],
+            select   => [ 'category', 'state', { count => 'me.id' } ],
+            as       => [ qw/category state state_count/ ],
+        }
+    );
+
+    my %by_category = map { $_->category => {} } $c->stash->{contacts}->all;
+    my %recent_by_category = map { $_->category => 0 } $c->stash->{contacts}->all;
+
+    my $state_map = {};
+
+    $state_map->{$_} = 'open' foreach FixMyStreet::DB::Result::Problem->open_states;
+    $state_map->{$_} = 'closed' foreach FixMyStreet::DB::Result::Problem->closed_states;
+    $state_map->{$_} = 'fixed' foreach FixMyStreet::DB::Result::Problem->fixed_states;
+    $state_map->{$_} = 'scheduled' foreach ('planned', 'action scheduled');
+
+    while (my $p = $problems->next) {
+        my $meta_state = $state_map->{$p->state};
+        $by_category{$p->category}->{$meta_state} += $p->get_column('state_count');
+    }
+
+    $params = {
+        %area_param,
+        'me.confirmed' => { '>=', $time },
+    };
+
+    my $recent_problems = $c->model('DB::Problem')->to_body(
+        $c->stash->{body}
+    )->search(
+        $params,
+        {
+            group_by => [ 'category' ],
+            select   => [ 'category', { count => 'me.id' } ],
+            as       => [ qw/category category_count/ ],
+        }
+    );
+
+    while (my $p = $recent_problems->next) {
+        $recent_by_category{$p->category} += $p->get_column('category_count');
+    }
+
+
+    $c->stash->{$_} = 0 for values %$state_map;
+    $c->stash->{open} = $c->model('DB::Problem')->to_body(
+        $c->stash->{body}
+    )->search(
+        {
+            %area_param,
+            confirmed => { '>=' => DateTime::Format::W3CDTF->format_datetime($date) },
+        }
+    )->count;
+
+    while (my $comment = $comments->next) {
+        my $meta_state = $state_map->{$comment->get_column('state')};
+        $c->stash->{$meta_state} += $comment->get_column('state_count');
```

Comment:
Not sure if this was missed: A comment problem_state is set (most of :-/ ) the time with an update, regardless of if there's been a state change or not; so I don't think it can be used as is as a statistic for the number of issues scheduled/closed/fixed in the past month :-/ You could perhaps go on staff users only? But not even then I imagine.
Inspect form now leaves an update; if the admin editing is similarly updated, then at least we'd have an update for every state change (after a month). But even then you'd have to go through the updates to know when one was a 'change' and when one wasn't changing it.